### DISCUSSION TITLE
Sticky nvim-tree and not using window_picker: How to remember the window where navigating into nvim-tree from?

### DISCUSSION THREAD

**Discussion by englut** [nvim-tree/nvim-tree.lua#2722] [Q&A]
I have it set up so `nvim-tree` is stickied open (as sticky as I can get it at least).  I also don't want to use window_picker.

What I'm experiencing: I'll open a window, so I have 2 open, one with a file I opened originally, and one empty new one I just created.... while inside the empty new window, I change windows to `nvim-tree`, and open a file... the file is opened in the first window rather than the one I was just in.

reading the documentation for disabling the window picker, I was confused by the wording:

```
Enable the feature. If the feature is not enabled, files will open in window from which you last opened the tree.
```

I'm assuming that it has to be the window when the tree *opened*, and not when you *navigated into* the tree from... in which case... does anyone know if there's an open to support a "stick" `nvim-tree`?

Apologies if this is not the right place for this kind of question, I can't seem to find any issues similar to what I'm experiencing.
I also went to the Matrix channel to post this, but looks like it's been pretty dead, so trying my luck here 🙏

**Comment by alex-courtis** [comment]
There are some issues regarding syncing of window and tab information #2255 however I'm not sure that's what we are seeing.

The doc is technically correct however the behaviour is not generally useful. When navigating to the tree via:
* a wincmd, say `<c-w>h`, the originating window will not be updated
* `api.tree.open` or similar, the originating window will be updated

I do use the latter rather than wincmds which results in the more predictable behaviour I think you desire.

See #2262
